Source/WebInspectorUI/UserInterface/Views/NetworkTableContentView.js

```diff
@@ -1396,7 +1396,7 @@ WI.NetworkTableContentView = class NetworkTableContentView extends WI.ContentVie
         if (!mainResource)
             return false;
 
-        if (!mainResource.requestSentDate)
+        if (!mainResource.hasMetadata)
```

Comment:
So I looked at `WI.HARBuilder` and I think technically the only thing that's actually required for a `WI.Resource` is the `requestSentDate`.  All of the other information (e.g. `timingData`, `remoteAddress`, etc.) is optional.

As such, I'd probably undo this change and leave it as it was before (or maybe instead create a `WI.Resource.prototype.hasHARData` or something).  My apologies for suggesting you change it.

Reply:
Done. Removed